python/visualize.py

```diff
@@ -142,7 +142,8 @@ def update_progress(n_intervals):
               Input('interval-component', 'n_intervals'))
 def show_status(input_value):
     if done:
-        df = beta_df[["name", "value"]]
+        df = beta_df[beta_df['target'] == False]
+        df = df[["name", "value"]]
         text = 'Finished! Final betas:'
         datatable = dash_table.DataTable(df.to_dict('records'),
                                          [{'name': i, 'id': i} for i in df.columns])
```

Comment:
The visualizer is obviously the best output of this work, but I wonder if I wonder if it shouldn't be kept separate (maybe a different directory?) Otherwise it feels a bit like mixing the debug code with the actual algorithm implementation

Reply:
Good point, I moved  them to separate dirs and added them to `.dockerignore`